templates/tabby-web.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>tabby-web</Name>
+  <Repository>ghcr.io/eugeny/tabby-web:latest</Repository>
+  <Registry>https://hub.docker.com/r/outlinewiki/outline/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/Eugeny/tabby-web/issues/new</Support>
+  <Project>https://tabby.sh</Project>
+  <Overview>Tabby Web - an SSH/Telnet/Serial client in your browser.&#xD;</Overview>
+  <Category>Productivity:</Category>
+  <WebUI>http://[IP]:[PORT:9090]</WebUI>
+  <Requires>MariaDB&#xD;</Requires>
+  <Config Name="Port HTTP" Target="80" Default="9090" Mode="tcp" Description="" Type="Port" Display="always" Required="true" Mask="false">9090</Config>
+  <Config Name="Database URL" Target="DATABASE_URL" Default="mysql://root:123@db/tabby" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">mysql://root:123@db/tabby</Config>
+  <Config Name="Debug" Target="DEBUG" Default="false" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">false</Config>
+  <Config Name="Docker Arguments" Target="DOCKERIZE_ARGS" Default="-wait tcp://db:3306 -timeout 60s" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">-wait tcp://db:3306 -timeout 60s</Config>
+  <Config Name="Port" Target="PORT" Default="9090" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">9090</Config>
```

Comment:
This is supposed to be the internal port 80: https://github.com/Eugeny/tabby-web/blob/abcdfbcd4e003168189d6698db901e50ca39a1c0/docker-compose.yml#L11

That said, since the "Port HTTP" variable allows users to edit the external port that maps to internal 80, I don't think we should allow users to potentially shoot themselves in the foot by changing the internal port. If there's a way we can skip declaring this variable, let's do it.

Reply:
Good catch. I will push the change now. 

Also, the application defaults to port 8000 when not setting the PORT variable. However, when remapping the "Port HTTP" from port 80 to port 8000, the Web UI doesn't show. As much as I would like to take the possibility of issues out of the picture, I think we might be stuck with setting the "PORT" variable to port 80 with a description stating, "DO NOT CHANGE - This sets the internal port for the Web UI".